src/ClientHandle.cpp

```diff
@@ -1952,7 +1953,7 @@ void cClientHandle::Tick(float a_Dt)
 	if (m_BlockDigAnimStage > -1)
 	{
 		int lastAnimVal = m_BlockDigAnimStage;
-		m_BlockDigAnimStage += (int)(m_BlockDigAnimSpeed * a_Dt);
+		m_BlockDigAnimStage += (int) (m_BlockDigAnimSpeed * a_Dt);
```

Comment:
You like your cast spaces :)

Why not make it a static_cast, since you've edited the line already?